continuous_integration/scripts/make-conda-env

```diff
@@ -6,17 +6,6 @@ make_ci_env() {
     mamba env create -n "${CONDA_ENV}" -f "${REPO_DIR}/continuous_integration/environment.yml"
```

Comment:
Please check if the python version can be set here by adding `python=${PYTHON_VERSION}`. 

```
mamba env create -n "${CONDA_ENV}" -f "${REPO_DIR}/continuous_integration/environment.yml" python=${PYTHON_VERSION}
```

Reply:
I did this in the follow up review. It seems to work, in that the conda env does have the appropriate python, however there is something which causes "which python" to be 3.13. Whether that be an environment variable issue, or otherwise I will debug it in the testing PR.